rpc/cloudagent/compute/baremetalmachine/moc_cloudagent_baremetalmachine.proto

```diff
@@ -45,34 +33,13 @@ message BareMetalMachineOperatingSystemConfiguration {
 	LinuxConfiguration linuxConfiguration = 6;
 }
 
-message BareMetalMachineNetworkInterface {
-	string networkInterfaceName = 1;
-}
-
-message BareMetalMachineNetworkConfiguration {
-	repeated BareMetalMachineNetworkInterface interfaces = 1;
-}
-
-message BareMetalMachineSize {
-	int32 cpuCount = 1;
-	int32 gpuCount = 2;
-	int32 memoryMB = 3;
```

Comment:
[Discussion] It might make sense to have size properties for BMM, and treat it as minimum requirements for the machine. That, cloud agent will choose the smallest BM host that meets the requirements.

It is fine if you want to punt on this for now, as we don't have clear requirements regarding BM host selection.

Reply:
Yes, I will punt this. We had discussed this as a future addition when the requirements are clear. Currently we choose the first BMH found.